test-server/test-echo.c

```diff
@@ -208,7 +211,10 @@ int main(int argc, char **argv)
 	int disallow_selfsigned = 0;
 #endif
 
+#ifdef WIN32
```

Comment:
Ah ok, well I see now. But I'd prefer you remove the `#ifdef` here, and instead move so that the `set_log_level` later is also used on windows (but not the syslog stuff). And while you're at it replace all  `WIN32` with `_WIN32`.